Services.Test/IotHub/IotHubConnectionStringManagerTest.cs

```diff
@@ -4,25 +4,31 @@
 using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Exceptions;
 using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.IotHub;
 using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Runtime;
+using Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Storage;
 using Moq;
 using Services.Test.helpers;
 using Xunit;
 
-namespace Services.Test
+namespace Services.Test.IotHub
 {
     public class IotHubConnectionStringManagerTest
     {
         private readonly Mock<ILogger> logger;
         private readonly Mock<IDiagnosticsLogger> diagnosticsLogger;
         private readonly IServicesConfig config;
+        private readonly Mock<IServicesConfig> mockConfig;
```

Comment:
nit: naming convention. Either add `mock` prefix to all mocks, or keep them consistent.